SeleniumWrapper/Utils/Browser.cs

```diff
@@ -5,12 +5,12 @@ public class Browser : IBrowser
     public WebDriverWait BrowserWait { get; set; }
     public WebDriver WebDriver { get; }
 
-    public DriverOptions Options { get; set; }
-
     public Browser(WebDriver webDriver)
```

Comment:
Class browser is not fully realized, where the rest methods, driver has more, rewatch the lesson recording